internal/testcontext/compile.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testcontext
+
+import (
+	"os/exec"
+	"path"
+)
+
+// Compile compiles the specified package and returns the executable name.
+func (ctx *Context) Compile(pkg string) string {
+	ctx.test.Helper()
```

Comment:
What does this do?

Reply:
This marks the function as a test helper. Which means that it isn't listed when a test fails and instead points to the nearest non-test-helper.

This means when compiling fails it will point to the line calling `ctx.Compile` instead of `ctx.test.Error(string(out))` inside this function.